batchflow/models/eager_torch/resnet.py

```diff
@@ -0,0 +1,361 @@
+        If int, then number of filters in the output tensor. Default value is 'same'.
+    kernel_size : int, list of int
+        Convolution kernel size. Default is 3.
+    strides : int, list of int
+        Convolution stride. Default is 1.
+    downsample : int, bool
+        If int, in first repetition of block downsampling with a factor `downsample`.
+        If True, in first repetition of block downsampling with a factor 2.
+        If False, without downsampling. Default is False.
+    bottleneck : bool, int
+        If True, then construct a canonical bottleneck block from the given layout.
+        If False, then bottleneck is not used. Default is False.
+    se : bool
+        If True, then construct a SE-ResNet block from the given layout.
+        If False, then squeeze and excitation is not used. Default is False.
+    groups : int
+        Use `groups` convolution side by side, each  seeing 1 / `groups` the input channels,
+        and producing 1 / `groups` the output channels, and both subsequently concatenated.
+        Number of `inputs` channels must be divisible by `groups`. Default is 1.
+    op : str or callable
+        Operation for combination shortcut and residual.
+        See more :class:`~.layers.Combine` documentation. Default is '+'.
+    n_reps : int
+        Number of times to repeat the whole block. Default is 1.
+    kwargs : dict
+        Other named arguments for the :class:`~.layers.ConvBlock`
+    """
+    def __init__(self, inputs=None, layout='cnacna', filters='same', kernel_size=3, strides=1,
+                 downsample=False, bottleneck=False, se=False, groups=1, op='+', n_reps=1, **kwargs):
+        #pylint: disable=eval-used
+        super().__init__()
+
+        num_convs = sum([letter in CONV_LETTERS for letter in layout])
+
+        vars = {key: get_num_channels(inputs) for key in ['S', 'same']}
+        filters = [filters] * num_convs if isinstance(filters, (int, str)) else filters
+        filters = [eval(str(item), {}, vars) for item in filters]
+
+        kernel_size = [kernel_size] * num_convs if isinstance(kernel_size, int) else kernel_size
+        strides = [strides] * num_convs if isinstance(strides, int) else strides
+        strides_d = list(strides)
+        groups = [groups] * num_convs
+        side_branch_stride = np.prod(strides)
+        side_branch_stride_d = int(side_branch_stride)
+
+        if downsample:
+            downsample = 2 if downsample is True else downsample
+            strides_d[0] *= downsample
+            side_branch_stride_d *= downsample
+        if bottleneck:
+            bottleneck = 4 if bottleneck is True else bottleneck
+            layout = 'cna' + layout + 'cna'
+            kernel_size = [1] + kernel_size + [1]
+            strides = [1] + strides + [1]
+            strides_d = [1] + strides_d + [1]
+            groups = [1] + groups + [1]
+            filters = [filters[0] // bottleneck] + filters + [filters[0]]
```

Comment:
Change it back as in the paper.
```suggestion
            filters = [filters[0]] + filters + [filters[0]  * bottleneck]
```